lib/src/main/java/com/auth0/jwt/algorithms/RSAAlgorithm.java

```diff
@@ -64,6 +63,19 @@ public void verify(DecodedJWT jwt) throws SignatureVerificationException {
         }
     }
     
+    @Override
+    public byte[] sign(byte[] contentBytes) throws SignatureGenerationException {
```

Comment:
shouldn't this method also be annotated as `@deprecated` ?

Reply:
@Deprecated added